Content.Server/GameObjects/Components/Mobs/StunnableComponent.cs

```diff
@@ -91,7 +91,10 @@ public override void Initialize()
         /// <param name="seconds">How many seconds the mob will stay stunned</param>
         public void Stun(float seconds)
         {
-            seconds = Math.Min(seconds + _stunnedTimer, _stunCap);
+            seconds = Math.Min(_stunnedTimer + (seconds * StunTimeModifier), _stunCap);
```

Comment:
What is this not `MathF.Min`?

Reply:
Good catch. Fixed now.

Reply:
There's not really a difference between `MathF.Min()` and `Math.Min()` so I feel like this didn't need changing but whatever.

Reply:
Oh... Well, the other methods were using MathF so I guess this was more of a consistency change--